src/features/MoviesPage/styled.js

```diff
@@ -0,0 +1,14 @@
+import styled from "styled-components";
+
+export const Wrapper = styled.section`
+    display: grid;
+    grid-template-columns: repeat(auto-fit, minmax(324px, 1fr));
```

Comment:
A jednak! Dobra robota 😄

Reply:
tak jest, mówiłem że to idealne rozwiązanie 😁